.claude/commands/gap-analysis.md

```
@@ -0,0 +1,141 @@
+| Feature | 94% | dashboard | /gap-analysis feature |
+| Platform | 90% | web fixes | /gap-analysis platform |
+
+**Next Step**: Run `/gap-analysis [layer]` for details, or `/gap-planning [layer]` to plan.
+```
+
+## Instructions
+
+### Step 1: Determine Output Type
+
+**Layer Parameters**:
+| Parameter | Template | Action |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)